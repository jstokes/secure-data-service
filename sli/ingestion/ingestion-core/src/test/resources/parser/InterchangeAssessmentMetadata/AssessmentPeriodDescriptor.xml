<?xml version="1.0" encoding="UTF-8"?>
<InterchangeAssessmentMetadata xmlns:sli="http://slc-sli/ed-org/0.1" xmlns="http://ed-fi.org/0100"
  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
  <AssessmentPeriodDescriptor id="ID001">
    <CodeValue>Ph.p</CodeValue>
    <Description>Q.3vp-iteadfawBE4nKNIM8nrIMqjx9c9ECfa_FEyFh8.du6SXZTUHxBdEb0ioS-AaJ3h:p8vswGncsABZt2IIrF_EvuMKL7aGg8v75b3IZcVF1N49-CqBVWoA8bp4qzJpcqrdSuHkqtKCms12rxc41gr9pjzv3JXwv5nJB4yYS5goq5:V4</Description>
    <ShortDescription>qErUBbhashENMUs-</ShortDescription>
    <BeginDate>2014-03-23</BeginDate>
    <EndDate>2014-12-03</EndDate>
  </AssessmentPeriodDescriptor>
</InterchangeAssessmentMetadata>
