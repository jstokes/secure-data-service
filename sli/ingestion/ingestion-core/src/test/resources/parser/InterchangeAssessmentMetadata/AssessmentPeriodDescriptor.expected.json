{
  "@id":"ID001",
  "CodeValue":{"_value":"Ph.p"},
  "Description":{"_value":"Q.3vp-iteadfawBE4nKNIM8nrIMqjx9c9ECfa_FEyFh8.du6SXZTUHxBdEb0ioS-AaJ3h:p8vswGncsABZt2IIrF_EvuMKL7aGg8v75b3IZcVF1N49-CqBVWoA8bp4qzJpcqrdSuHkqtKCms12rxc41gr9pjzv3JXwv5nJB4yYS5goq5:V4"},
  "ShortDescription":{"_value":"qErUBbhashENMUs-"},
  "BeginDate":{"_value":"2014-03-23"},
  "EndDate":{"_value":"2014-12-03"}
}